src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -8378,8 +8378,6 @@ namespace WaterThermalTanks {
             DataLoopNode::NodeData const &HPWHCondWaterOutletNode = DataLoopNode::Node(HeatPump.CondWaterOutletNode);
             Real64 const HPWHCondenserDeltaT = HPWHCondWaterOutletNode.Temp - HPWHCondWaterInletNode.Temp;
             Tank.SourceInletTemp = Tank.SourceOutletTemp + HPWHCondenserDeltaT;
-        } else {
-            Tank.SourceInletTemp = Tank.SourceOutletTemp;
```

Comment:
@nmerket Can you please review this change?  At first it seemed like a straightforward change, but now I am a bit lost on why any of the logic in here would ever be overwriting Tank.SourceInletTemp.  That should be fixed once at the beginning of the simulation, with a temperature provided by the plant, and then never touched.  Only outlets.  Thoughts?

Reply:
This is okay. The intention was that for wrapped condenser HPWHs, there is no source side heat transfer because it's handled separately. This was to ensure that `Qsource` was indeed zero for that case, but unintentionally applies it to other stratified tank configurations as well.